src/debug/matsuda_debug_menu.c

```diff
@@ -460,14 +300,172 @@ static void sub_80A9F50(void)
 
 static void sub_80A9FE4(void)
 {
-    u8 ptr[5];
-
-    memcpy(ptr, gMatsudaDebugMenu_UnknownByteArray, 5);
+    u8 ptr[] = _("{HIGHLIGHT WHITE2} ");
     DmaFill32Large(3, 0, (void *)VRAM, VRAM_SIZE, 0x1000);
     sub_80034D4((void *)VRAM, ptr);
     LoadFontDefaultPalette(&gWindowTemplate_81E6C3C);
 }
 
+//const u8 gMatsudaDebugMenu_UnknownByteArray[] = {0xFC, 0x02, 0x0F, 0x00, 0xFF};
+const u8 Unknown_083C922D[] = INCBIN_U8("graphics/unknown/ball.4bpp");
+const u16 Unknown_083C924E[] = INCBIN_U16("graphics/unknown/ball.gbapal");
+
+const u8 gUnknown_083C926E[][2] =
+    {
+        {2,  0},
```

Comment:
Why did you put extra indent spaces here?

Reply:
no clue where that came from tbqh. might have been my IDE trying to do what it thought was cleaning it up?